webdriver-spec.html

```diff
@@ -2191,7 +2191,7 @@
        in <var>matched capabilities</var> to <var>capability value</var>.
 
      <dt>"<code>pageLoadStrategy</code>"
-     <dd><p>If <var>capability value</var> is not a string,
+     <dd><p>If <var>capability value</var> is not in the <a>page load strategies table</a>,
```

Comment:
This also needs to highlight which column to look at in said table. Building on what Simon said above, I would suggest the following wording:

> If capability value is not a keyword in the table of page load strategies,